css/amp.css

```diff
@@ -167,6 +167,16 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+.i-amphtml-fade-out .i-amphtml-blur {
+  opacity: 0 !important;
+  transition: opacity 1s 1s !important;
```

Comment:
Why have a [transition-delay](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-delay) of 1s? Also, let's use `ease` timing function.

`transition: opacity 1s ease !important;`

Reply:
Yeah I did it because I thought it looked better on fast connections where the image looked like it flashed since the placeholder wasn't seen. But the 'ease' function solves that problem so thanks!

Reply:
I'm not aware of other AMP animation curves, but the material guidelines would recommend a "decelerating curve" here, either `ease-out` or the actual material easing: `cubic-bezier(0.0, 0.0, 0.2, 1)`.